src/agent/configuration_parser/src/configuration_parser.cpp

```diff
@@ -9,6 +11,11 @@ namespace configuration
 {
     ConfigurationParser::ConfigurationParser()
     {
+        if(!std::filesystem::exists(CONFIG_FILE_NAME))
+        {
+            GenerateDefaultConfig();
```

Comment:
I think this function is not necessary. If the file doesn't exist, the agent should fail and exit execution.

Reply:
Removed.